crates/matrix-sdk-base/src/client.rs

```diff
@@ -93,10 +93,10 @@ pub struct BaseClient {
     /// The store used for encryption
     #[cfg(feature = "e2e-encryption")]
     crypto_store: Arc<dyn CryptoStore>,
-    /// The olm-machine that is created once the [`matrix_sdk::Session`] is set
-    /// via [`BaseClient::restore_login`]
+    /// The olm-machine that is created once the [`crate::session::Session`] is
```

Comment:
```suggestion
    /// The olm-machine that is created once the [`Session`][crate::session::Session] is
```

same below. `crate` probably shouldn't be part of the rendered text.